packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -176,12 +176,8 @@ contract TaikoL1 is
         });
     }
 
-    function getBlockFee(uint32 gasLimit) public view returns (uint64) {
-        return LibUtils.getBlockFee({
-            state: state,
-            config: getConfig(),
-            gasAmount: gasLimit
-        });
+    function getBlockFee() public view returns (uint64) {
```

Comment:
Shall we also change the calculation of L2 EIP-1559? since its "purchasing" the `block.gaslimit`.

Reply:
If `block.gaslimit` is set to `blockMaxGasUsed` I think it's fine.